### ISSUE TITLE
Can't open GUI for moving players

### ISSUE THREAD

**Issue by AnonAlpha** [GeyserMC/Geyser#696]
**Describe the bug**

If a player is moving rapidly, the GUI which should open won't show up
**To Reproduce**

1. Use what ever menu plugin
2. move rapidly when trying to open a gui
3. see error

**Expected behavior**

GUI open as usual
**Screenshots / Videos**

None
**Server Version**

Paper version git-Paper-322

**Geyser Version**

Geyser pull request for inventory fix #602 
**Minecraft: Bedrock Edition Version**

1.14.6 all platform
**Additional Context**

**Comment by towuk118** [comment]
hi, the fact is that a visual chest is created over the player and if you move away from the chest for several blocks, it will disappear!

**Comment by AnonAlpha** [comment]
> hi, the fact is that a visual chest is created over the player and if you move away from the chest for several blocks, it will disappear!

Yes I'm aware of that, but this is a issue I think, for many plugins need to make player open GUI when moving, even open GUI when player are high up in the sky where y cord is higher than 256, Camotoy said there's a solution by using minecart chest, maybe talk about this with him?

**Comment by Heath123** [comment]
Fixed by #708

**Comment by Jens-Co** [comment]
I'll close this issue since the pr that was made for it got closed. See; https://github.com/GeyserMC/Geyser/pull/708#issuecomment-1018995568
